code/blocks/PromoBlock.php

```diff
@@ -47,8 +47,8 @@ public function getCMSFields()
 
         if ($this->ID) {
             $config = GridFieldConfig_RelationEditor::create();
-            if (class_exists('GridFieldSortableRows')) {
-                $config->addComponent(new GridFieldSortableRows('SortOrder'));
+            if (class_exists('GridFieldOrderableRows')) {
```

Comment:
Do we assume the requirement of this extension in our composer file and provide documentation supporting for manual installations? I this moving forward it's a push towards package management which should remove the need for `class_exists()` calls.